### ISSUE TITLE
[Bug]: Phone number not visible in profile even though saved in profile

### ISSUE THREAD

**Issue by aurisnoctis** [nextcloud/server#47610]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

1. phone number can be entered and is saved in profile (persistently visible when editing profile) ✔️ 
2. phone number is **not** shown in profile view (when not in edit mode) ❌ 
3. phone number **is** shown in system address book ✔️ 

How can the phone number be made visible in the profile?

Found Issue ["Phone number can't be set at user profile #25931"](https://github.com/nextcloud/server/issues/25931) but that seems to be a separate issue.

Thanks in advance 👍 

### Screenshots for the 3 points above
![2024-08-22_171935_firefox](https://github.com/user-attachments/assets/1c2009e0-0367-4682-998b-61200c48dff6)
_________________________________________
![2024-08-22_171810_firefox](https://github.com/user-attachments/assets/99f7571c-b901-45a5-85e7-1f5daa1dbea4)
_________________________________________
![2024-08-22_171756_firefox](https://github.com/user-attachments/assets/43fcdb1a-6973-4ea5-95b6-1938281a58c2)

### Steps to reproduce

1. go to https://<myncserver>/index.php/settings/user and set a phone number
2. make sure that the scope directly set above the field is set to at least "local" = visible to people of the same instance and guests
3. make sure that at the bottom of the profile page, the profile visibility for "phone" is set to "show to logged in accounts only"
4. exit saved profile --> phone symbol is visible in user profile but the phone number is missing, see https://<myncserver>/index.php/u/<put your user name here>
5. open the system address book at https://<myncserver>/index.php/apps/contacts/ where ONLY the system address book is active and check my own entry: phone number is there
6. wonder why phone number is not shown in own profile for other users but just in system phone book

### Expected behavior

The phone number should be visible in my profile.

### Nextcloud Server version

29

### Operating system

Other

### PHP engine version

Other

### Web server

Other

### Database engine version

Other

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
Our instance is a managed Nextcloud with Webo.
```

### List of activated Apps

```shell
Our instance is a managed Nextcloud with Webo. Copy-Paste from the admin GUI pages of "Active apps":

Active apps - All apps are up-to-date.
	
Activity	2.21.1 	Featured
Calendar	4.7.16 	Featured
Collaborative tags	1.19.0 	Featured
Comments	1.19.0 	Featured
Contacts	6.0.0 	Featured
Dashboard	7.9.0 	Featured
Deck	1.13.1 	Featured
Deleted files	1.19.0 	Featured
Federation	1.19.0 	Featured
File reminders	1.2.0 	Featured
File sharing	1.21.0 	Featured
Files download limit	2.0.0 	Featured
First run wizard	2.18.0 	Featured
Forms	4.2.4 	Featured
Group folders	17.0.2 	Featured
Guests	3.1.0
Log Reader	2.14.0 	Featured
Mail	3.7.7 	Featured
Nextcloud announcements	1.18.0 	Featured
Notifications	2.17.0 	Featured
ONLYOFFICE	9.3.0 	Featured
Password policy	1.19.0 	Featured
PDF viewer	2.10.0 	Featured
Photos	2.5.0 	Featured
Polls	7.2.1
Privacy	1.13.0 	Featured
Recommendations	2.1.0 	Featured
Related Resources	1.4.0 	Featured
Share by mail	1.19.0 	Featured
Support	1.12.0 	Featured
Talk	19.0.8 	Featured
Teams	29.0.0-dev 	Featured
Text	3.10.1 	Featured
Two-Factor TOTP Provider	11.0.0-dev 	Featured
Update notification	1.19.1 	Featured
Usage survey	1.17.0 	Featured
User status	1.9.0 	Featured
Versions	1.22.0 	Featured
Weather status	1.9.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Additional info

Our Nextcloud version 29.0.5 (Nextcloud Hub 8) is hosted by Webo. I don't have access to the admin console, which is why I couldn't supply all information asked for above.

**Comment by aurisnoctis** [comment]
Hi @Anubi5x0 I don't have access to the server - it is hosted for us by a company. I can however ask the hoster to provide certain info via a support ticket if necessary to understand this problem I described.

**Comment by aurisnoctis** [comment]
The phone number (or mail address or whatever) is **not shown** after the respective symbol. **But 📞 itself is a button** that can be clicked or copied, a `tel:` link.

It certainly looks to me like the phone number is missing after the 📞 symbol, see screenshots above, because the **mail address is shown** after the ✉️ symbol. 👀

**Comment by joshtrichards** [comment]
> Just realized this confusing point: The phone number (or mail address or whatever) is not shown after the respective symbol. But 📞 itself is a button that can be clicked or copied, a tel: link. So the phone number is not totally missing in the profile.

Yes, only the icons are displayed there. Same goes for email, etc. Tooltips show the details.

> However, it certainly looks to me like the phone number should be displayed after the 📞 symbol, see screenshots above, because the mail address is shown after the ✉️ symbol. 👀

I see in your screenshot where this is the case, but I can't reproduce any scenario where more than the email icon is visible in Nc 25-29. 

Anyhow, as to whether more than the icons should be visible here, that's a design question so I'll label this issue appropriately and let designers weigh in.

**Comment by susnux** [comment]
After reviewing this request and considering community feedback, it appears there hasn't been significant interest or engagement from other users.

We appreciate your suggestion and understand the value you see in it. However, at this time, we will not be pursuing its implementation and the current way (only the link as an icon) was done on purpose / design.

Thank you for your understanding.
